### PR TITLE
use maven package to make a zip file for make dist 

### PR DESCRIPTION
**Pull request by glorysdj**
This change is [Reviewable](https://reviewable.io/reviews/intel-analytics/bigdl/768)

### CODE REVIEW

**Comment by glorysdj** [comment]
pass the validation at
http://172.168.2.101:8080/view/PR-Validation/job/BigDL-PR-Validation/588/

File: dist.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          <include>bigdl.sh</include>
+          <include>classes.lst</include>
+          <include>img_class.lst</include>
+      </includes>
+    </fileSet>
+    <fileSet>
+      <outputDirectory>/lib</outputDirectory>
+      <directory>spark/dl/target</directory>
+      <includes>
+          <include>bigdl-*-jar-with-dependencies.jar</include>
```
Comment (yiheng) [inline dist.xml]: please also add the dependencies-and-spark jar. I notice in some example, some data preprocess job is still standalone java program. We still need that jar

**Comment by yiheng** [comment]
can you also update the make-dist.sh script?

**Comment by glorysdj** [comment]
both changes are done and pass the pr-validation:http://10.239.47.211:18888/view/PR-Validation/job/BigDL-PR-Validation/624

**Comment by yiheng** [comment]
I don't think we should delete the make-dist.sh
1. it will check java/maven version for user
2. it appears a lot in our document. delete it need all theses document need to be updated
3. we just need to remove the copy logic in the scripts and copy the zip to the root folder?

File: make-dist.sh [hunk 2]
```diff
@@ -47,30 +47,3 @@ fi
 
 mvn clean package -DskipTests $*
```
Comment (yiheng) [inline make-dist.sh]: please unzip to dist folder to keep consisted with the original build doc

**Comment by yiheng** [comment]
please squash these commits

**Pull request merged**